### DISCUSSION TITLE
Login via Código

### DISCUSSION THREAD

**Discussion by LuizLivio** [wppconnect-team/wppconnect#2121] [General]
Existe algum exemplo de como utilizar o catchLinkCode para logar por código ao invés de QRCode? Só encontrei o commit com a implementação da feature, porém não encontrei nenhum exemplo. Tentei utilizar, porém não entra no callback para exibir o código no console.

Abri como uma discussão e não como issue porque não sei se existe algum problema ou somente se eu não estou sabendo usar.

**Comment by LuizLivio** [comment]
Alguem?

**Reply by fereinaux** [reply]
quando descobrir avisa kkkk faz meses que sempre que tenho um tempo livre tento implementar
